Tasks/Gradle/CodeAnalysis/pmd.gradle

```diff
@@ -0,0 +1,31 @@
+rootProject {
+    afterEvaluate {project ->
+        def hasPmd = false;
+
+        allprojects.forEach{ p ->
+
+            if (p.plugins.hasPlugin('pmd')) {
```

Comment:
If any project has pmd enabled - do not try to apply the plugin. We'll still try to process the results.